### PR TITLE
Added Numbers Game Play | 947

### PR DESCRIPTION
**Pull request by byte-king**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

This is the numbers game.
You have to guess the number in the given number range in 5 moves, If you correctly guess it, you win else you will lose.

Motivation: Took motivation from Binary Search and wanted to use React hooks in real life projects.

Dependencies: No new dependencies required

Fixes # (issue) https://github.com/reactplay/react-play/issues/947

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

### CODE REVIEW

**Comment by byte-king** [comment]
@atapas  @koustov

File: src/plays/the-numbers-game/styles.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/* enter stlyes here */
+.play-details {
```
Comment (koustov) [inline src/plays/the-numbers-game/styles.css:2]: Use play specific class name 
e.g.: `number-games-play-details`

File: src/plays/the-numbers-game/styles.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 10px;
+  padding: 10px;
+  border-width: 2px;
+  color: #100f0f;
+  border-radius: 14px;
+}
+input {
```
Comment (koustov) [inline src/plays/the-numbers-game/styles.css:12]: Do not use pseudo selectors. Use specific class selectors.

File: src/plays/the-numbers-game/styles.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  border-color: #271a1a;
+  background-color: #9a9aa4;
+  color: #100f0f;
+  border-style: solid;
+  border-radius: 14px;
+  box-shadow: 0px 0px 4px rgba(66, 66, 66, 0.31);
+  text-shadow: 0px 0px 1px rgba(155, 195, 193, 0.83);
+}
+
+.buttons {
```
Comment (koustov) [inline src/plays/the-numbers-game/styles.css:53]: Use play specific name

File: src/plays/the-numbers-game/styles.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+}
+input::placeholder {
+  color: #817979;
+  font-style: italic;
+}
+input:focus {
+  outline: none;
+}
+
+Button {
```
Comment (koustov) [inline src/plays/the-numbers-game/styles.css:36]: Do not use pseudo selectors

File: src/plays/the-numbers-game/styles.css [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  border-width: 2px;
+  border-color: #271a1a;
+  background-color: #9a9aa4;
+  color: #100f0f;
+  border-style: solid;
+  border-radius: 14px;
+  box-shadow: 0px 0px 4px rgba(66, 66, 66, 0.31);
+  text-shadow: 0px 0px 1px rgba(155, 195, 193, 0.83);
+}
+input::placeholder {
```
Comment (koustov) [inline src/plays/the-numbers-game/styles.css:28]: Do not use pseudo selectors

File: src/plays/the-numbers-game/styles.css [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  color: #817979;
+  font-style: italic;
+}
+input:focus {
```
Comment (koustov) [inline src/plays/the-numbers-game/styles.css:32]: Do not use pseudo selectors

File: src/plays/the-numbers-game/styles.css [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+  box-shadow: 0px 0px 4px rgba(66, 66, 66, 0.31);
+  text-shadow: 0px 0px 1px rgba(155, 195, 193, 0.83);
+}
+
+.buttons {
+  display: flex;
+  justify-content: center;
+}
+
+.moves {
```
Comment (koustov) [inline src/plays/the-numbers-game/styles.css:58]: Use play specific names

File: src/plays/the-numbers-game/TheNumbersGame.js [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+import './styles.css';
+import { useState, useEffect } from 'react';
+import 'react-toastify/dist/ReactToastify.css';
+import { ToastContainer, toast } from 'react-toastify';
+
+// WARNING: Do not change the entry componenet name
+function TheNumbersGame(props) {
+  // Your Code Start below.
+  const [guess, setGuess] = useState('');
+  // const [message, setMessage] = useState('Guess a number in the given range');
```
Comment (koustov) [inline src/plays/the-numbers-game/TheNumbersGame.js:11]: Please remove commented code

**Review by koustov — CHANGES_REQUESTED** [review]
- Please try to reduce the cover and background image size. 
- I have given some tiny comments

File: src/plays/the-numbers-game/bg.png
Comment (koustov) [inline src/plays/the-numbers-game/bg.png:1]: I don't see this image being used.

**Review by koustov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**